src/main/java/mods/eln/Eln.java

```diff
@@ -5206,31 +5492,33 @@ private void recipeElectricalSource() {
     }
 
     private void recipeElectricalCable() {
-        addRecipe(signalCableDescriptor.newItemStack(1),
-            "R",
-            "C",
+        addRecipe(signalCableDescriptor.newItemStack(2), //signal wire
+            "R", //rubber
+            "C", //iron cable
+			"C",
             'C', findItemStack("Iron Cable"),
             'R', "itemRubber");
 
-        addRecipe(lowVoltageCableDescriptor.newItemStack(1),
+        addRecipe(lowVoltageCableDescriptor.newItemStack(2), //Low Voltage Cable
             "R",
             "C",
+			"C",
```

Comment:
May want to check with @Baughn that this is alright for upstream too. Otherwise it will be really confusing.

Reply:
I very much want to avoid merging anything that adds indentation errors.

Reply:
Still not indented properly.